interface/orders/orders_results.php

```diff
@@ -34,13 +34,13 @@
 // Check authorization.
 $thisauth = acl_check('patients', 'med');
 if (!$thisauth) {
-    die(xl('Not authorized'));
+    die(xlt('Not authorized'));
```

Comment:
This is correct use of xlt.
As an aside, the xlt function is basically shorthand for:
```
htmlspecialchars(xl(), ENT_NOQUOTES)
```